packages/mdc-toolbar/mdc-toolbar.scss

```diff
@@ -61,12 +69,13 @@ $mdc-toolbar-mobile-breakpoint: 599px;
       justify-content: flex-end;
       order: 1;
     }
+  }
 
-    .mdc-toolbar__title {
-      @include mdc-typography(title);
+  &__title {
+    @include mdc-typography(title);
 
-      margin: 0;
-    }
+    margin: 0;
+    line-height: 1.5rem;
```

Comment:
Yes. But that line-height don't fits this case. The default line height was 2rem.